### DISCUSSION TITLE
Display freezing after Filament Runout

### DISCUSSION THREAD

**Discussion by mosheas** [Jyers/Marlin#101] [Q&A] [answered]
Hi, nice job, thank you, I think i fund some bug.. so after filament Runout Happens, and when park nozzle is time out, i change the filament , and reprint than happens. the Display gets freezing on message "Reheat Finished Press To Continue" , even though the printer continues to print. only turning it off and on again it's fixes the phenomenon.

**Comment by Jyers** [comment, marked as answer]
This is fixed in the latest release.

**Comment by mosheas** [comment]
Yes, thank you, is it's work fine, Your firmware works great, thank you
very much.
